wallet/src/plan.rs

```diff
@@ -306,6 +223,8 @@ where
     }
 
     Ok(plans)
+    */
+    unimplemented!("This function has not been implemented since we changed the swap mechanism to auto-claim in scanning.")
```

Comment:
Followup: restore implementation of manually claiming unclaimed swaps

This is necessary if the command fails partway through for some reason. We may have to extend the view protocol to return a list of all unclaimed swaps to do so.  We should also restore the hook to claim unclaimed swaps in the `tx sweep` implementation, which is currently commented out.